src/Resource.ts

```diff
@@ -38,6 +38,10 @@ export class Resource extends BaseResource {
     return this.model.name
   }
 
+  public idName(): string {
```

Comment:
not 100% sure if we need this. You can also do `const primaryKey = this.properties().find((p) => p.isId())`

Reply:
it seems easier to save one time, rather than computing it each time with a `find` call?